src/llvm-api.cpp

```diff
@@ -176,37 +176,6 @@ LLVMExtraCreateFunctionPass(const char *Name, jl_value_t *Callback)
     return wrap(new JuliaFunctionPass(Name, Callback));
 }
 
-class JuliaBasicBlockPass : public BasicBlockPass {
-public:
-    JuliaBasicBlockPass(const char *Name, jl_value_t *Callback)
-        : BasicBlockPass(CreatePassID(Name)), Callback(Callback)
-    {
-    }
-
-    bool runOnBasicBlock(BasicBlock &BB)
-    {
-        jl_value_t **argv;
-        JL_GC_PUSHARGS(argv, 2);
-        argv[0] = Callback;
-        argv[1] = jl_box_voidpointer(wrap(&BB));
-
-        jl_value_t *ret = jl_apply(argv, 2);
-        bool changed = jl_unbox_bool(ret);
-
-        JL_GC_POP();
-        return changed;
-    }
-
-private:
-    jl_value_t *Callback;
-};
-
-extern "C" JL_DLLEXPORT LLVMPassRef
-LLVMExtraCreateBasicBlockPass(const char *Name, jl_value_t *Callback)
```

Comment:
CC: @maleadt

Reply:
I guess it's a bit sudden to get rid of it like that, but apparently the BB pass is broken and I don't use it, so :man_shrugging:

Reply:
Maybe sudden, but been deprecated for a while though.